index.html

```diff
@@ -2,8 +2,8 @@
 <html>
 
 <body>
-  <h1>Hello World</h1>
```

Comment:
Hey why did you remove this?

Reply:
Oh, that's a good point. I didn't mean to do that.

Reply:
Will update the pull request now.